gnomad_constraint/utils/constraint.py

```diff
@@ -403,6 +411,8 @@ def apply_models(
         are considered well covered and was used to build plateau models. Sites
         below this cutoff have low coverage and was used to build coverage models.
         Default is `COVERAGE_CUTOFF`.
+    :param low_coverage_filter: Lower median coverage cutoff for coverage filter. Sites with coverage n below this cutoff will
+        removed from `exome_ht` and 'context_ht'.
```

Comment:
Same thoughts on this being confusing with cov_cutoff

```suggestion
    :param low_coverage_filter: Lower median coverage cutoff for coverage filter. 
        Sites with coverage below this cutoff will be removed from`exome_ht` and 
        'context_ht'.
```